File: app/src/main/java/com/mai/nix/maiapp/room/AppDatabase.java
package com.mai.nix.maiapp.room;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.mai.nix.maiapp.model.SubjectBody;
import com.mai.nix.maiapp.model.SubjectHeader;

@Database(entities = {SubjectHeader.class, SubjectBody.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract SubjectsDao subjectsDao();
}

File: app/src/main/java/com/mai/nix/maiapp/database/AppDatabase.kt
package com.mai.nix.maiapp.database

import androidx.room.Database
import androidx.room.RoomDatabase
import com.mai.nix.maiapp.model.Day
import com.mai.nix.maiapp.model.ExamModel
import com.mai.nix.maiapp.model.Subject

@Database(entities = [Day::class, Subject::class, ExamModel::class], version = 1)
abstract class AppDatabase: RoomDatabase() {
    abstract val scheduleDao: ScheduleDao
    abstract val examDao: ExamsDao
}